packages/react/src/styles.scss

```diff
@@ -149,7 +149,7 @@ $deprecations--message: 'Deprecated code was found, this code will be removed be
 @import 'components/PaginationNav/pagination-nav';
 @import 'components/UIShell/ui-shell';
 @import 'components/List/carbon-list';
-@import 'components/Table/TableBody/TableBodyLoadMoreRow/table-body-load-more-row';
+@import 'components/ResourceList/resourcelist';
```

Comment:
Why is the `components/Table/TableBody/TableBodyLoadMoreRow/table-body-load-more-row` removed`?

Reply:
Fixed it.